packages/mdc-chips/chip/foundation.js

```diff
@@ -60,6 +60,9 @@ class MDCChipFoundation extends MDCFoundation {
       notifyRemoval: () => {},
       getComputedStyleValue: () => {},
       setStyleProperty: () => {},
+      hasLeadingIcon: () => {},
+      getBoundingClientRect: () => {},
```

Comment:
I think adding `root` in the method makes it more clear. `getRootBoundingClientRect`

Reply:
Done